4.analyze_data/notebooks/UMAP/scripts/0.UMAP_coordinates.py

```diff
@@ -157,27 +160,47 @@
         if plate_name == "Plate_3_prime":
             df["Metadata_Plate"] = "Plate_3_prime"
 
+        # For Plate_4, only include rows where Metadata_siRNA is "None"
+        if plate_name == "Plate_4":
+            # Fill NaN values in "Metadata_siRNA" column with "No Construct"
+            df["Metadata_siRNA"].fillna("No Construct", inplace=True)
+
+            # Filter out rows where "Metadata_siRNA" is "No Construct"
+            df = df[df["Metadata_siRNA"] == "No Construct"]
```

Comment:
I'll change my code to account for this

Reply:
👍 Sounds good! I am realizing in future projects we should make sure any `None` values are something else since Python and Pandas defaults it to a Nan.